common-pipeline/operators/uihints-v3-schema.json

```diff
@@ -597,6 +597,15 @@
                   "type": "number"
                 }
               }
+            },
+            "tooltipDirection": {
+              "description": "Action image tooltip direction. Default is bottom.",
```

Comment:
You can specify a default value in the schema instead of mentioning the default in the description

Reply:
This should also be in snake_case instead of camalCase.